addons/account/views/account_move_views.xml

```diff
@@ -332,7 +332,8 @@
                     <separator/>
                     <filter string="To Check" name="to_check" domain="[('move_id.to_check', '=', True)]"/>
                     <separator/>
-                    <filter string="Unreconciled" domain="[('amount_residual', '!=', 0), ('account_id.reconcile', '=', True)]" help="Journal items where matching number isn't set" name="unreconciled"/>
+                    <filter string="Unreconciled" domain="[('account_id.reconcile', '=', True)]" name="unreconciled"/>
```

Comment:
This spec really doesn't make sense to me... an Unreconciled filter will display reconciled items? maybe a help string will be useful?

Reply:
Yeah i agree this filter is weird but it was asked by Fabien, even benj is not a fan of that but the spec was: 
"unreconciled, show all items whith the property ALLOW RECONCILIATION, no matter if residual is =0 ou !=0"